components/app-core/backend/main.go

```diff
@@ -312,7 +312,10 @@ func loadPortalConfig(pc interfaces.PortalConfig) (interfaces.PortalConfig, erro
 
 func loadDatabaseConfig(dc datastore.DatabaseConfig) (datastore.DatabaseConfig, error) {
 	log.Debug("loadDatabaseConfig")
-	if err := config.Load(&dc); err != nil {
+
```

Comment:
Had a look and also talked to Irfan, it's not something we can do at the moment. This is called right at the beginning of the PP start up process before the portal proxy object and plugins are initialised. Fixing this would require a bit of a rework.